template/ios/Podfile

```diff
@@ -1,19 +1,19 @@
+
 require_relative '../node_modules/react-native/scripts/react_native_pods'
 require_relative '../node_modules/@react-native-community/cli-platform-ios/native_modules'
 
 platform :ios, '10.0'
 
-target 'HelloWorld' do
-  # Pods for HelloWorld
-  use_react_native!
+target 'RNTestProject' do
```

Comment:
I guess these `RNTestProject` strings should be reverted, right?

Reply:
ah right! Copy paste XD